be/src/storage/rowset/segment_v2/page_decoder.h

```diff
@@ -33,6 +35,15 @@ class Column;
 namespace starrocks {
 namespace segment_v2 {
 
+struct PageCacheOptions {
+    fs::ReadableBlock* rblock = nullptr;
+    bool save_in_page_cache = false;
+    bool kept_in_memory = false;
```

Comment:
FYI: you can put these two bool variables at the last to reduce the size of the struct.